### ISSUE TITLE
Whitepaper to provide perspectives and background  on data models refering to Material Recovery/Recycling 

### ISSUE THREAD

**Issue by tkMXSS** [eclipse-tractusx/eclipse-tractusx.github.io#926]
### Description
The Whitepaper will provide insights on the current state of technology of material recovery for different materials from end-of-life vehicles. The process descriptions start at the dismantler and follow through up the manufacturer, who mixes recycled material with prime to produce new automotive parts. The information in the paper touch upon economic, technical, regulatory or environmental (e.g. PCF) requirements and aim to portrait a balanced view between opportunities and challenges associated with commercial, industrial scale material recovery from ELVs. The purpose of the document is to provide sufficient background information for the corresponding data models required for the different materials, which potentially can differ depending on the process and material in question.  Corresponding data models could be stemming (but not limited to) material accounting. Linking the data models and potentially further developing ones will be performed in a future step.   

Corresponding author: Roland Ballus (thyssenkrupp Material Servics), [roland.ballus@thyssenkrupp-materials.com](mailto:roland.ballus@thyssenkrupp-materials.com)
Participants: Michael Georg Schmidt (BASF),  Pauline Holstein (brainofmaterials), Frederic Tholence (Volvo cars),  Maurizio Griva (Reply), Ernesto Barceló Rodriguez (Gestamp), Christiane Grünewald (TÜV SÜD)

### Impact
The whitepaper could be added to the Circularity KIT.

### Additional information

- [ ]  I'm willing to contribute to this feature

**Comment by stephanbcbauer** [comment]
@stanfaldin is this an open-source feature? So, will this end up in a kit?

**Comment by stanfaldin** [comment]
> @stanfaldin is this an open-source feature? So, will this end up in a kit?

Yes, will be integrated as a Link, supported by a short description
